03-create.md

```diff
@@ -6,7 +6,8 @@ minutes: 10
 ---
 > ## Learning Objectives {.objectives}
 > 
-> *   Create a local Git repository.
+> *   Learn how to create a local Git repository using `git init`.
```

Comment:
I prefer the original version: we are writing the learning objectives as "In this topic, you will learn to ..."